packages/backend/src/graphql/project/queries/projectsQueryField.ts

```diff
@@ -48,7 +43,7 @@ builder.queryField('projects', (t) =>
                       showProjectsForOtherUser,
                     ),
                     // check if the given user is allowed to see the project
-                    showProjectsForOtherUser ? getWhereUserIsMember(projectMemberUserId.toString()) : {},
+                    showProjectsForOtherUser ? getWhereUserIsMember(userIds?.[0]?.toString() ?? '') : {},
```

Comment:
this is an error, it must calculate all userIds, not only the first one.

Make all Intersections (AND) and Unions (OR) very clear for the reviewer